Manifest.toml

```diff
@@ -49,6 +49,12 @@ git-tree-sha1 = "5b08ed6036d9d3f0ee6369410b830f8873d4024c"
 uuid = "b99e7846-7c00-51b0-8f62-c81ae34c0232"
 version = "0.5.8"
 
+[[CSDP]]
```

Comment:
CSDP is a bad dependency to have because it does not build on windows.
Can you use SCS?
Also this means we should be testing tutorial in all OSs.

Reply:
Ok, I don't know why it was used, I was surprised this even didn't error before

Reply:
Automatic tests are only testing linux

Reply:
removed CSDP from the repo

Reply:
Did you push the changes? Looks like it's still here.

Reply:
indeed, I just did, had forgotten -_-